desktop/src/main/java/bisq/desktop/primary/main/content/components/ChatMessagesComponent.java

```diff
@@ -308,8 +313,18 @@ private void privateTradeMessagesChanged(BisqEasyPrivateTradeChatChannel chatCha
                     model.getSendBtcAddressButtonVisible().set(walletService.isPresent());
                 }
             });
+            twoPartyPeerLeft(chatChannel);
+        }
+
+        private void twoPartyPeerLeft(PrivateChatChannel<?> chatChannel) {
```

Comment:
As its also called from `privateTradeMessagesChanged(BisqEasyPrivateTradeChatChannel chatChannel)` the method name `twoPartyPeerLeft` is not optimal, the BisqEasyPrivateTradeChatChannel is a groupChannel not a 2-party channel.

Reply:
Also its not the case that the peer left, just that the leaveChannelButton visible state gets updated. Maybe call it `updateLeaveChannelButtonState`?